src/lib/y2storage/autoinst_profile/partition_section.rb

```diff
@@ -431,6 +432,14 @@ def lvm_group_name(device)
       def fixed_size?(device)
         device.is?(:disk_device, :software_raid)
       end
+
+    private
+
+      def include_partition_type?(partition)
```

Comment:
Please, document the method even if it is private. That documentation will help your teammates if they need to check your code :) In this case it may seems obvious, but we have `Planned::Partition` and `Partition` classes (and `PartitionSection` too) so it is nice the have the arguments annotated.